dbhydra/dbhydra_core.py

```diff
@@ -497,11 +496,15 @@ def generate_table_dict(self):
 
 
 class BigQueryDb(AbstractDB):
-    def __init__(self, credentials_path, project_id, dataset_name):
+    def __init__(self, db_details):
         # super().__init__(None,None)
-        self.credentials_path = credentials_path
-        self.project_id = project_id
-        self.dataset = dataset_name
+        self.credentials_path = db_details["DB_SERVER"]
+        # self.project_id = project_id
+        self.dataset = db_details["DB_DATABASE"]
+
+        self.locally = True
+        if db_details["LOCALLY"] == "False":
+            self.locally = False
```

Comment:
`self.locally = db_details.get("LOCALLY", True)` ?

Reply:
this solution works only if db_details["Locally"] is boolean, but in past it included string "False" or "True", so it was needed to convert it to boolean

Reply:
Maybe we have to discuss whether not to add LOCALLY flag to API. I would set it to False in Big Query for now, because the db reflection has to set now either true or false and it then doesnt work properly.

Reply:
hmmm, in fact, we need to discuss the locally option. I believe you wont be able to access local db from API. That means we should disallow `locally` option for web app. @DovaX

Reply:
I agree @tobyy22. But please make a task in Jira (if it's still on the table - don't know, it's been 3 weeks) so we don't forget.

Reply:
@tobyy22